CHANGES.rst

```diff
@@ -13,6 +13,19 @@ Changelog
 
 .. towncrier release notes start
 
+4.3.4 (2022-04-01)
+Bugfixes
+--------
+
+- Consume proxy authentication fix in pulp_ansible 0.7.4
+  `AAH-840 <https://issues.redhat.com/browse/AAH-840>`_
```

Comment:
Since this didn't actually fix the issue, can we add a note here or clarification about that?

Reply:
Updated :)